packages/components/package.json

```diff
@@ -47,11 +47,11 @@
     "dialog-polyfill": "^0.5.6",
     "ember-a11y-refocus": "^3.0.2",
     "ember-auto-import": "^2.6.3",
-    "ember-cached-decorator-polyfill": "^0.1.4",
-    "ember-cli-babel": "^7.26.11",
+    "ember-cached-decorator-polyfill": "^1.0.2",
+    "ember-cli-babel": "^8.2.0",
```

Comment:
[question] do we understand what the impact will be for consumers like cloud-ui who are still on 7.x?

Reply:
we understand it [in theory](https://github.com/emberjs/ember-cli-babel/blob/master/CHANGELOG.md#boom-breaking-change) (it's made of a `brocolli-babel-transpiler` bump and a node version drop), but it's a good call to see how they resolve in our consumer's projects (will do just that). if it gets ugly we might need to postpone some of them

Reply:
Tested this branch in our consumers' codebases.

The likes of `cloud-ui` resolve them well, `atlas` (and any other products that use `dependency-lint`) requires a few bumps or, alternatively, the following manual resolution:
```
    "@ember/test-waiters": "3.0.2",
    "ember-cached-decorator-polyfill": "0.1.4",
    "ember-composable-helpers": "4.5.0",
    "ember-element-helper": "0.6.1",
```